### ISSUE TITLE
headerImage

### ISSUE THREAD

**Issue by tspeidel** [digitalcraftsman/hugo-type-theme#29]
Beautiful theme - thanks for sharing.
I can't figure out how to add a headerImage (sorry - Hugo newbie here). 
Using one of the images included for testing which I copied to static/img:

```
	# To add a header background to the main site, save your image under static/img/.
	# If no image path is provided, a orange background will be shown instead.
	headerImage = "static/img/sample_feature_img.png"
```

I've tried various combinations of paths, but I can't get it to display a custom image.

**Comment by amedeedaboville** [comment]
I just installed this theme and looked it up as well.
The source says it goes to 
```
<div  class="call-out" 
style="background-image: url('{{ .Site.BaseURL }}/img/{{ .Site.Params.headerImage }}')"
>
```
So it looks like you need `headerImage = "sample_feature_img.png"`, and this worked on my site.

**Comment by amedeedaboville** [comment]
Additionally you need to have a headerText be not empty or the clause doesn't kick in :/
I left mine as `headerText = " "`

**Comment by digitalcraftsman** [comment]
@tspeidel as @amedeedaboville already mentioned you've to under `static/img/yourimage.png` and add `headerImage = "yourimage.png"` to the config file. I tried to clarify this in https://github.com/digitalcraftsman/hugo-type-theme/commit/390905671947209bd885815c9adb3b17fe0e039f.

@amedeedaboville in https://github.com/digitalcraftsman/hugo-type-theme/commit/25651775e3eddaa5e1b31ad0edfcd33e86556950 I made the headerImage independent from headerText. This way you don't have to use the workaround with the space in headerText.
